packages/rule-tester/tests/eslint-base/eslint-base.test.js

```diff
@@ -913,14 +944,28 @@ describe("RuleTester", () => {
         }, /fatal parsing error/iu);
     });
 
-    it("should not throw an error if invalid code has at least an expected empty error object", () => {
+    it("should throw an error if an error object has no properties", () => {
+      assert.throws(() => {
         ruleTester.run("no-eval", require("./fixtures/no-eval"), {
-            valid: ["Eval(foo)"],
-            invalid: [{
-                code: "eval(foo)",
-                errors: [{}]
-            }]
+          valid: ["Eval(foo)"],
+          invalid: [{
+            code: "eval(foo)",
+            errors: [{}]
+          }]
```

Comment:
[Refactor] The upstream uses 4 spaces for indentation. Could you please copy-paste it without re-indenting it? (we try to keep this file as close to the upstream as possible :slightly_smiling_face:)